### PR TITLE
update list and start comparison table

### PR DESCRIPTION
**Pull request by almereyda**
This PR contains a commit which

* renders the unordered list into a table for starting to work on #7 
* adds solidarity hosters from #12 
* adds otherwise related contacts from #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,57 +2,71 @@
...
-  - [rhien - Réseau d'Hébergeurs Indépendants et ENgagés](http://www.rhien.org/Hebergeurs,59.html)
-  - [A/I - Links and Friends - Radical Tech Collectives](http://www.autistici.org/en/links.html)
-
-* wiki documents using git and markdown
-
-  - [Digital Tools for Activism](https://github.com/drewrwilson/toolsforactivism)
-  - [Awesome-Selfhosted](https://github.com/Kickball/awesome-selfhosted)
-
-## License
+## groups and collectives which provide solidarity IT services
```
Comment (pierreozoux) [inline README.md]: I'd put `Groups` (start titles with uppercase)

**Comment by pierreozoux** [comment]
Nice :) Please just update the comment and merge it yourself :)

**Comment by almereyda** [comment]
I have provided said commit, but cannot merge for myself, as I am not presented any option for this.

**Comment by pierreozoux** [comment]
You have to accept my invitation to be admin of the repo :)

**Comment by almereyda** [comment]
Didn't get any. Probably lost within the myriads of emails piling up every day.

**Pull request merged**